test/suite/test_config07.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2018 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import fnmatch, os
+import wiredtiger, wttest
+from wtscenario import make_scenarios
+
+# test_config07.py
+#    Test that log files extend as configured and as documented.
+class test_config07(wttest.WiredTigerTestCase):
+    uri = "table:test"
+    entries = 100000
+    K = 1024
+
```

Comment:
I suggest replacing this variable with  `log_size = ` and then you can use the same log size in the `expected_log_size` definitions and the connection configuration so that if we want to test other log sizes, only one place needs to change.

Reply:
I may not be able to replace `K=1024` with `log_size =` but I have added a new variable `log_size`. This is because `K` will be required by the `expected_log_size` definitions of the valid cases like `log_extend_len='(log=100K)'`.